core/src/main/scala/cats/effect/SyncConsole.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+
+import java.io.{ BufferedReader, PrintStream }
+
+import cats.Show
+import cats.syntax.show._
+
+/**
+  * A default instance for any `F[_]` with a `Sync` instance.
+  *
+  * Construct one e.g. by calling `SyncConsole.stdIn[F]`.
+  */
+private class SyncConsole[F[_]](private val out: PrintStream,
```

Comment:
I have OCD when it comes to this style :laughing: Can we make it consistent with the others? If you introduce an Enter before open parens and another before closing parens `scalafmt` will pick the style.